ansys/dpf/core/examples/examples.py

```diff
@@ -19,3 +19,4 @@
 msup_transient = os.path.join(_module_path, "msup_transient_plate1.rst")
 simple_cyclic = os.path.join(_module_path, "file_cyclic.rst")
 distributed_msup_folder = os.path.join(_module_path, 'msup_distributed')
+ASME_SecVIII_Div2 = os.path.join(_module_path, 'ASME_SecVIII_Div2.rst')
```

Comment:
we try to not push result files in this repo to keep our package as light as possible. Could you please refer to [this ](https://github.com/pyansys/pydpf-core/pull/179#discussion_r893067628) comment?

Reply:
I followed Ayush example